elad_testing/hash_pickles.py

```diff
@@ -0,0 +1,6 @@
+import pickle
```

Comment:
Please put any test-related auxiliary scripts and modules inside the `tests` folder. An even fi you create a sub-directory there, it can't be called Elad. :)

That's not informative enough. Contributors should know what functionality is this related to.

Reply:
will remove this file